requirements.txt

```diff
@@ -0,0 +1,29 @@
+aniso8601==8.1.0
```

Comment:
Your README says you're using poetry, but it looks like this was using pip? Instead we should make sure the requests library has been added to your pyproject.toml so that the instructions still work as expected